docs/algob-console.md

```diff
@@ -16,4 +16,117 @@ Following globals are available in an `algob console` REPL:
 * `deployer` : algob deployer in run mode. User can access checkpoints, get logic signature, transferAlgos and all other functions supported by `algob deployer`.
 * `algodClient` : `algosdk.Algodv2`- an instance of algorand driver based on the current network (default if `--network` flag is not passed).
 * `algosdk` : User can access `algosdk` package functions using this object. (eg. `algosdk.encodeAddress(..)`)  
-* `algob` : all`algob` [exported](https://github.com/scale-it/algorand-builder/blob/master/packages/algob/src/index.ts) functions (eg. `algob.mkAccounts(..)`, `algob.balanceOf(..) etc)`
\ No newline at end of file
+* `algob` : all`algob` [exported](https://github.com/scale-it/algorand-builder/blob/master/packages/algob/src/index.ts) functions (eg. `algob.mkAccounts(..)`, `algob.balanceOf(..) etc)`
+
+# Example Walkthrough
+
+This section demonstrates a detailed walkthrough of executing `/scripts/transfer` functions in `examples/deployment` using `algob console`. The project used for this walkthrough is `examples/deployment` where user is able to 
+* transfer `algos` between accounts
+* transfer `assets` (eg. `gold`, `tesla`)
+* transfer `algos` directly from smart contract(contract account) or by delegating authority.
+
+using `algob console`.
+
+## Setup
+
+In the  `examples/deployment` directory:
+1. Follow the README file
+2. deploy the assets and smart contracts:
+```
+yarn run algob deploy
+``` 
+This will deploy your assets (`gold` and `tesla` in this case) and store asc1 `logic signature` in checkpoint for delegated approval mode. Also, let's initialize some of the accounts which we will use in further transactions. 
```

Comment:
```suggestion
This will deploy your assets (`gold` and `tesla` in this case) and store asc1 `logic signature` in cp for delegated approval mode. Also, let's initialize some of the accounts which we will use in further transactions. 
```